docs/setup/new_environment.md

```diff
@@ -230,6 +230,20 @@ $ commcare-cloud monolith deploy-stack --skip-check -e 'CCHQ_IS_FRESH_INSTALL=1'
 $ commcare-cloud monolith fab deploy
 ```
 
+The first time you run deploy (and whenever we have changes to our couch indices), it will kick off a "preindex" task. The deploy script will block until this task is complete. If it takes too long, the deploy script will pause, with a message:
```

Comment:
>  we have changes to our couch indices

I think `preindex_everything` also initializes the ES indicies when they don't exist.